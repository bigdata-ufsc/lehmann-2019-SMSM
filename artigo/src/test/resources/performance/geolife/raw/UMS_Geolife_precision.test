{
	"dataset": {
		"name" : "geolife_university",
		"raw": 'true',
		"params": {
			"pointsTable" : "geolife.geolife_inside_university_5_pois",
			"stopTable" : "stops_moves.geolife_inside_university_stop_5_pois",
			"moveTable" : "stops_moves.geolife_inside_university_move_5_pois"
		}
	},
	"groundtruth": {
		"type": "String",
		"index": 10
	},
	"measure": {
		"name": "UMS"
	}
}