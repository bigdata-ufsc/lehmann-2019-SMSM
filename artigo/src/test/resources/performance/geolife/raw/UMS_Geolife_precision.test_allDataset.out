Executing SQL...
Fetching...
Loaded 102314 GPS points from database
Loaded 337 trajectories from database
Semantic Trajectories statistics: mean - 298,10, min - 15,00, max - 887,00, sd - 154,83
New dataset size: 331
Semantic Trajectories statistics: mean - 288,92, min - 15,00, max - 732,00, sd - 140,12
[331] Elapsed time 83202 miliseconds
